gen/generator.go

```diff
@@ -1655,6 +1699,39 @@ func ignoreProperty(propertyName, classPkgName string, definitions Definitions)
 	return false
 }
 
+func ignoreTestProperty(propertyName, classPkgName string, definitions Definitions) (bool, string) {
+
+	precedenceList := []string{classPkgName, "global"}
+	for _, precedence := range precedenceList {
+		if classDetails, ok := definitions.Properties[precedence]; ok {
+			for key, value := range classDetails.(map[interface{}]interface{}) {
+				if key.(string) == "ignore_properties_in_test" {
+					for k, v := range value.(map[interface{}]interface{}) {
```

Comment:
If we go by what I suggested in properties.yml the assertion will likely change here.
We would just return true or false for this function